doc/examples/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -110,9 +110,11 @@ int main(int argc, char** argv)
   {
     RCLCPP_FATAL(LOGGER, "Exception while creating planning plugin loader %s", ex.what());
   }
+
+  const auto& planner_name = planner_plugin_names.at(0);
```

Comment:
I know this is just a tutorial, but if for some reason the parameter exists BUT is an empty vector, this will segfault. I think in the spirit of not making this super complex, maybe just add this as a warning in a comment above this line.